src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -145,6 +152,12 @@ public ApiClient load( Class<?> key ) throws Exception {
         usedByESID = entitySetsApi.getEntitySetId( USED_BY_ENTITY_SET_NAME );
         participatedInESID = entitySetsApi.getEntitySetId( PARTICIPATED_IN_AESN );
         userAppsESID = entitySetsApi.getEntitySetId( CHRONICLE_USER_APPS );
+        questionnaireESID = entitySetsApi.getEntitySetId( QUESTIONNAIRE_ENTITY_SET_NAME );
+        questionsESID = entitySetsApi.getEntitySetId( QUESTIONS_ENTITY_SET_NAME );
+        answersESID = entitySetsApi.getEntitySetId( ANSWERS_ENTITY_SET_NAME );
+        addressesESID = entitySetsApi.getEntitySetId( ADDRESSES_ENTITY_SET_NAME );
+        respondsWithESID = entitySetsApi.getEntitySetId( RESPONDS_WITH_ENTITY_SET_NAME );
+        partOfESID = entitySetsApi.getEntitySetId( PART_OF_ENTITY_SET_NAME );
```

Comment:
I think it's time...

```
// EdmConstants
public static final ImmutableSet<String> ENTITY_SET_NAMES = ImmutableSet.of( 
    STUDY_ENTITY_SET_NAME,
    DATA_ENTITY_SET_NAME,
    ...
);

// ChronicleServiceImpl
Map<String, EntitySet> entitySets = entitySetsApi.getEntitySetsByName( ENTITY_SET_NAMES );
studyESID = entitySets.get( STUDY_ENTITY_SET_NAME ).getId();
deviceESID = entitySets.get( DEVICES_ENTITY_SET_NAME ).getId();
...
```

Reply:
I agree.  Should this be done separately though since it requires refactoring a bunch of other places?